app/src/main/java/com/example/keniphhomemaintenance/maintenance_screen/MaintenanceItem.kt

```diff
@@ -1,8 +1,15 @@
 package com.example.keniphhomemaintenance.maintenance_screen
 
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "maintenance_item_table")
 data class MaintenanceItem(
+    @PrimaryKey(autoGenerate = true)
+    val maintenanceItemID: Int,
```

Comment:
Usually you just call this "id", since it's inherently a MaintenanceItemId due to it being an attribute on the MaintenanceItem class